src/_js/map.js

```diff
@@ -181,6 +181,43 @@
 			}
 		});
 	};
+	TibiaMap.prototype._loadMarkers = function () {
+		const _this = this;
+		const icons = []
+		const symbols = ["!", "$", "?", "bag", "checkmark", "cross", "crossmark", "down", "flag", "lock", "mouth", "red down", "red left", "red right", "red up", "skull", "spear", "star", "sword", "up"];
+		symbols.forEach(s => {
+			icons[s] = L.icon({iconSize: [11, 11], iconUrl: s.replace("!", "exclamation").replace("$", "dollar").replace("?", "question").replace(" ", "_") + ".png"});
+		});
+
+		const xhr = new XMLHttpRequest();
+		xhr.open('GET', 'https://tibiamaps.github.io/tibia-map-data/markers.json', true);
+		xhr.responseType = 'json';
+		xhr.onload = function () {
+			if (xhr.status === 200) {
+				xhr.response.forEach(m => {
+					let options = {'title': m.description};
```

Comment:
```suggestion
					const options = {'title': m.description};
```

([`const` is not about immutability!](https://mathiasbynens.be/notes/es6-const))